cif_pow.dic

```diff
@@ -121,6 +121,1229 @@ save_pd_amorphous.phase_id
+    _units.code                   angstrom_squared
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_background.Debye_diffuse_distance
+
+    _definition.id                '_pd_background.Debye_diffuse_distance'
+    _definition.update            2023-01-09
+    _description.text
+;
+    The value of the distance in a Debye diffuse scattering equation
+    representing the background in a calculated diffractogram.
+
+    The background equation is of the form
+
+        bkg = amplitude * sinc(distance * q) * exp(-displacement * q^2^)
+
+    where sinc(X) is defined as
+
+        sinc(X) = sin(X) / X
+
+    and where q is the magnitude of the diffraction vector, defined as
+
+        q = 4 * Pi * sin(theta) / lambda
+
+    where theta is the diffraction angle and lambda is the wavelength of the
+    incident radiation in angstroms.
+;
+    _name.category_id             pd_background
+    _name.object_id               Debye_diffuse_distance
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   angstroms
+
+save_
+
+save_pd_background.Debye_diffuse_distance_su
+
+    _definition.id                '_pd_background.Debye_diffuse_distance_su'
+    _definition.update            2023-01-09
+    _description.text
+;
+    Standard uncertainty of _pd_background.Debye_diffuse_distance.
+;
+    _name.category_id             pd_background
+    _name.object_id               Debye_diffuse_distance_su
+    _name.linked_item_id          '_pd_background.Debye_diffuse_distance'
+    _units.code                   angstroms
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_background.diffractogram_id
+
+    _definition.id                '_pd_background.diffractogram_id'
+    _definition.update            2023-01-08
+    _description.text
+;
+    A diffractogram id to which the background equation relates.
+;
+    _name.category_id             pd_background
+    _name.object_id               diffractogram_id
+    _name.linked_item_id          '_pd_diffractogram.id'
+    _type.purpose                 Link
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_background.id
+
+    _definition.id                '_pd_background.id'
+    _definition.update            2023-01-08
+    _description.text
+;
+    An arbitrary code identifying part of a background equation.
+;
+    _name.category_id             pd_background
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Word
+
+save_
+
+save_pd_background.line_segment_intensity_1
```

Comment:
Line segments could alternatively be presented as a loop where the key is a child of `_pd_data.point_id` and the other column is the intensity (plus silent key diffractogram.id). This suggestion assumes that you wouldn't want more than one such line segment sequence per diffractogram. New category could be named e.g. `PD_BACKGROUND_SEGMENT`

Reply:
I'll have a think on that.

Reply:
I think that keeping them here is OK, but just make it a single X,Y pair per packet, rather than two pairs making a line segment. That reduces data repetition and, as long as you have more than one coordinate pair, will be a valid set of line segments.

Reply:
You suggestion assumes that the X-coordinate of a segment corresponds to a data point, which mich not necessarily be the case.